### DISCUSSION TITLE
Run Workflow (Urgent)

### DISCUSSION THREAD

**Discussion by mweel1** [community/community#42164] [Actions]
### Select Topic Area

Bug

### Body

When I select Run Workflow in actions a list of branches never show up it just spins.

I get a 404 error when viewing the network traces to 

https://github.com/<acct>/<project>/actions/manual?workflow=.github%252Fworkflows%252Fstaging.yml

**Comment by persis-randolph** [comment]
Same issue here.

**Comment by m-davison** [comment]
I'm getting the same issue.  Can't load the Run Workflow dialog, and get 404

**Comment by markdb314** [comment]
Same issue - this is preventing our deployments.  Definitely an urgent issue 🚨🚨🚨

I tried running a workflow manually in two separate GitHub accounts with two totally different repos and got the same result

**Comment by renny-oh** [comment]
please, ASAP.. manual input workflow blocks business process.

**Comment by rcpeene** [comment]
Same issue

**Comment by scull7** [comment]
Same issue.  We are blocked from deploying code until this is resolved.

**Comment by scull7** [comment]
This is also reported here: https://github.com/orgs/community/discussions/42166

**Comment by ejdanderson** [comment]
Also experiencing this and preventing deployment.

**Comment by szofar** [comment]
Suggestion for those blocked: 
Temporarily set your workflow to run on a schedule instead of manually running it. Use a UTC time value 5 minutes from now or something like that.

```
# Controls when the workflow will run
on:
  # Triggers the workflow on cron schedule
  schedule:
    - cron: "0 12 * * *"
```

FWIW: seeing the same issue

**Comment by brayansdt** [comment]
Same error here...

**Comment by papunsenapati07** [comment]
https://www.githubstatus.com/ - it seems the status is updated now, expecting this to be fixed quickly.

**Comment by papunsenapati07** [comment]
The issue is resolved now - https://www.githubstatus.com/incidents/3y3y8nnbyt2d @mweel1
